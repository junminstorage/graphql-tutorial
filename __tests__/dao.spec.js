'use strict';

const dao = require('../src/dao');

describe('dao', () => {
    it('dao.projects', () => {
        return expect(dao.projects().then(d => typeof d)).resolves.toBe('object');
    });
    it('dao.tweet', () => {
        return dao.project(1)
            .then(p => {
                expect(p).toMatchSnapshot();
            });
    });
});
