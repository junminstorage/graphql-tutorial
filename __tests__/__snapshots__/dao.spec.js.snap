// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dao dao.tweet 1`] = `
Object {
  "author_id": 10,
  "body": "Lorem Ipsum",
  "date": 2018-01-01T00:00:00.000Z,
  "id": 1,
}
`;
